app/scripts/models/form-prefill.js

```diff
@@ -18,7 +18,8 @@ define([
     defaults: {
       email: null,
       password: null,
-      year: null
+      year: null,
```

Comment:
Can you make a note that `year` can be removed if we remove the dropdown based COPPA flow?